base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
+To make the best use of this guide, you should have a working knowledge of:
+
+* [Git](https://git-scm.com/): The version control system used.
+* [GitHub](https://github.com/thegooddocsproject): The project hosting platform used, along with associated project management tools.
+* [GitHub Flavored Markdown](https://github.github.com/gfm/): The wiki format used to store template documentation.
+* [Currently implemented doctypes templates](https://github.com/thegooddocsproject/templates): To help identify if you need to create a new template or edit an existing one.
+* Technical writing: To produce a well-written document that includes the relevant references.
+
+## Target audience for templates
+
+The templates target multiple audiences. They include writing tips for the following roles:
+
+
+### Template author
+
+{Template author tip: Remove this “template author” section from about-{doctype}-template.md files.}
+
+A template author creates a new template in line with base and core templates.
+
+
+### Template reviewer
+
+{Template author tip: Remove this “template reviewer” section from about-{doctype}-template.md files.}
+
+A template reviewer reviews the new template against template guidelines.
+
+
+### Template customizer
+
+A template customizer customizes a doctype template to align with a specific project or organizational guidelines. For example, specify the style guide to use with the template.
+
+
+### Docset owner
+
+A docset owner establishes an Information Architecture and Documentation Management Plan for a project. Typical tasks include:
+
+* Select a set of doctype templates for the project.
+* Customize templates if needed.
+* Select a style guide for the project.
+* Establish a docset maintenance processes.
+
+A docset owner should capture documentation decisions within a Documentation Management Plan.
+
+_{TBD: We should define a template for a Documentation Management Plan, then reference from here.}_
+
+The [Information Architecture Guide](https://thegooddocsproject.dev/ia-guide.html) aims to help docset owners with some of these tasks.
+
+### Document author
+
+A document author writes content in line with templates. Typically the document author is a subject matter expert and not a dedicated technical writer.
+
+Ideally, their document will receive feedback and guidance from a document reviewer and/or their docset owner.
+
+### Document reviewer
+
+Documentation is almost always improved when it is reviewed against the doctype’s template, the selected style guide, writing best practices and is reviewed for technical accuracy. While not essential, it helps if one of the reviewers is a dedicated technical writer.
+
+### Document reader
+
+And of course, we need to understand the needs of the final document reader. There are likely multiple reader personas to consider, which depends upon the reader’s role, their technical expertise, their specific goal, and so on.
+
+## Inheritance of template sets
```

Comment:
On Thu, Jan 7, 2021 at 4:44 AM T... wrote:
Hi Cameron, I checked out the base template. From what I understand, this is a template to provide a starting point for other templates? That's very meta but I can see the point. Overall, the sections were a bit confusing for me to follow. It wasn't clear to me which sections were intended as part of the template inheritance or not. For example, the "Inheritance of Template Sets" section -- is this part of the foundation for a template or just high-level information about templates, etc? The sections need to be clearer, in my opinion.

Reply:
Simplified personas, and split the docs to help make things easier to understand.